usaspending_api/awards/v2/lookups/elasticsearch_lookups.py

```diff
@@ -27,22 +27,22 @@
 
 base_mapping = {
     "Award ID": "display_award_id",
-    "Recipient Name": "recipient_name",
-    "Recipient DUNS Number": "recipient_unique_id",
-    "recipient_id": "recipient_unique_id",
-    "Awarding Agency": "awarding_toptier_agency_name",
-    "Awarding Agency Code": "awarding_toptier_agency_code",
-    "Awarding Sub Agency": "awarding_subtier_agency_name",
-    "Awarding Sub Agency Code": "awarding_subtier_agency_code",
-    "Funding Agency": "funding_toptier_agency_name",
-    "Funding Agency Code": "funding_toptier_agency_code",
-    "Funding Sub Agency": "funding_subtier_agency_name",
-    "Funding Sub Agency Code": "funding_subtier_agency_code",
-    "Place of Performance City Code": "pop_city_code",
+    "Recipient Name": "recipient_name.keyword",
```

Comment:
Talked: 
- should have had this `.keyword` on for all these. 
- would have just not matched queries against these fields' values (now it will)
- would need to do a deeper analysis on which of these are used for full-text search and which aren't